aep/general/0231/aep.md.j2

```diff
@@ -126,24 +115,66 @@ message BatchGetBooksResponse {
 }
 ```
 
-- The response message **must** include one repeated field corresponding to the
-  resources being retrieved.
+- The response message **must** include one repeated field where its items may
+  either be a retrieved resource or an error structure describing an error that
+  occurred attempting to retrieve the resource. The error alternative would
+  only be present for non-transactional batch gets.
 
 {% tab oas %}
 
-- The response schema **must** be an array with each item containing one of the
-  requested resources.
+- The response schema **must** be `type: object` with a single array property
+  with each item containing one of the requested resources.
+
+```json
+{
+  "results": [
+    {
+      "name": "publishers/lacroix/books/les-mis",
+      "isbn": "978-037-540317-0",
+      "title": "Les Misérables",
+      "authors": ["Victor Hugo"],
+      "rating": 9.6
+    },
+    {
+      "code": "NotFound",
```

Comment:
Should this be `404` rather than `"NotFound"`?  Or if not, where are these codes defined?  Is there a standard, or is it up to each BatchGet method to define its error codes?

Reply:
Should have been a ProblemDetails. Fixed.